### ISSUE TITLE
[$250] Invoices - Invoice receiver is unable to see Category, Tag and Tax fields in invoice report

### ISSUE THREAD

**Issue by IuliiaHerets** [Expensify/App#47579]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** v9.0.21-1
**Reproducible in staging?:** Y
**Reproducible in production?:** Y 
**If this was caught during regression testing, add the test name, ID and link from TestRail:** Exp https://expensify.testrail.io/index.php?/tests/view/4864174
**Email or phone of affected tester (no customers):** applausetester+kh050806@applause.expensifail.com
**Issue reported by:** Applause Internal Team

## Action Performed:
Precondition:
- Workspace has Categories, Tags and Taxes.

1. Go to staging.new.expensify.com
2. Go to FAB > Send invoice.
3. Enter amount. select user and send invoice.
4. Click on the invoice preview to go to transaction thread.
5. Enter data for Category, Tag and Tax.
5. As invoice receiver, go to invoice report.

## Expected Result:
Invoice receiver should be able to see Category, Tag and Tax in the transaction thread.

## Actual Result:
Invoice receiver is unable to see Category, Tag and Tax in the transaction thread. Only the system message for those fields are present for them.

## Workaround:
Unknown

## Platforms:

- [x] Android: Native
- [x] Android: mWeb Chrome
- [x] iOS: Native
- [x] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [x] MacOS: Desktop

## Screenshots/Videos

https://github.com/user-attachments/assets/7e2b75fc-622c-48db-aee0-3791eb58b144

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01e7476f563c3f9669</li>
        <li>Upwork Job ID: 1826516629658939965</li>
        <li>Last Price Increase: 2024-09-26</li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @johnmlee101</details>

**Comment by IuliiaHerets** [comment]
We think that this bug might be related to #vip-bills

**Comment by IuliiaHerets** [comment]
@RachCHopkins  FYI I haven't added the External label as I wasn't 100% sure about this issue. Please take a look and add the label if you agree it's a bug and can be handled by external contributors

**Comment by RachCHopkins** [comment]
Oh mine is _much_ more broken than this. 

![image](https://github.com/user-attachments/assets/d8b157ac-3dd0-433d-a731-67462b233711)

But despite it's terrible dupe display thing, I don't have the OP's problem.

![image](https://github.com/user-attachments/assets/f2ad1826-b415-41de-8626-251f88d3a63b)

**Comment by RachCHopkins** [comment]
@daledah I don't know how invoices should work on New Expensify, but typically the coding (category & tag) are irrelevant to the recipient and used only for internal purposes. However the tax is really important as they may need to claim it back. 

I would have thought that we should also not be showing the sender changing the coding, that's kinda private data. 

Is there someone who is 100% sure on how this is supposed to work?

**Comment by RachCHopkins** [comment]
Confirmed internally @daledah - the coding data on an invoice should be private to the company that sends it. It should not show on the recipient end, nor should any comment/history pertaining to the coding i.e. Category or Tag. The recipient should see the tax component and tax rate.

**Comment by RachCHopkins** [comment]
@Pujan92 how are you feeling about this proposal?

**Comment by Pujan92** [comment]
When user **A** sends invoice from workspace **W** to user **B** without adding them to the workspace **W**, user **B** won't have that policy data. That leads to the `isPolicyExpenseChat` false as @daledah mentioned.

I am not sure whether the BE should provide the policy data for user **B** or not in this case as mostly all fields are derived based on policy conditions. Adding a stamp for the internal engineer assignment and confirmation.

🎀👀🎀 C+ reviewed

**Comment by johnmlee101** [comment]
I'll take a look today!

**Comment by johnmlee101** [comment]
Sorry, I've been side-tracked massively by some other things, I'm hoping to look at this by EOW otherwise I'll find someone who has open time

**Comment by RachCHopkins** [comment]
@johnmlee101 What's the plan, Stan?

**Comment by johnmlee101** [comment]
Looking at this today

**Comment by johnmlee101** [comment]
Ugh, no update since last, been torn. if we need to unblock on this let me know and I'll see if someone can take it over

**Comment by RachCHopkins** [comment]
I don't think this is urgent. Invoicing functionality is not widely used as yet, so I don't think there's a particular urgency here.

**Comment by RachCHopkins** [comment]
Changing to Monthly - more realistic at this time of year.

**Comment by RachCHopkins** [comment]
@johnmlee101 do you think you will have capacity to pick this up again?

**Comment by RachCHopkins** [comment]
Bill pay is on hold indefinitely. Closing for now.
